packages/contracts/test/exchange/match_orders.ts

```diff
@@ -651,13 +651,7 @@ describe('matchOrders', () => {
             });
             // Match orders
             return expectRevertReasonOrAlwaysFailingTransactionAsync(
-                matchOrderTester.matchOrdersAndVerifyBalancesAsync(
-                    signedOrderLeft,
-                    signedOrderRight,
-                    takerAddress,
-                    erc20BalancesByOwner,
-                    erc721TokenIdsByOwner,
-                ),
+                exchangeWrapper.matchOrdersAsync(signedOrderLeft, signedOrderRight, takerAddress),
```

Comment:
I had to update this because the new version of `expectRevertReasonOrAlwaysFailingTransactionAsync` is more strict about the types of promises it accepts.

Reply:
Does it still verify the balances?